<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View graph</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="DotWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="refreshToolButton">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="zoomInToolButton">
        <property name="text">
         <string>Zoom in</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="zoomOutToolButton">
        <property name="text">
         <string>Zoom out</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="fitToolButton">
        <property name="text">
         <string>Best fit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="originalSizeToolButton">
        <property name="text">
         <string>100%</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>18</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="interactiveCheckBox">
        <property name="toolTip">
         <string>Auto refresh graph when the tree is changed</string>
        </property>
        <property name="text">
         <string>Interactive</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="groupCheckBox">
        <property name="toolTip">
         <string>Display only input/output groups</string>
        </property>
        <property name="text">
         <string>Groups</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DotWidget</class>
   <extends>QWidget</extends>
   <header>neuropype.gui.xdot_qt</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
